RFCs/RFC5.md

```diff
@@ -0,0 +1,16 @@
+# RFC5: Finances
+
+## Current vision for The Carpentries finances:
+* Software Carpentry (SWC) and Data Carpentry (DC) are currently separate financial entities with independent budgets, 
+both sheltered by our fiscal sponsor NumFOCUS. 
+* Most income for both organizations is derived from organizational membership agreements (e.g., contracts signed 
+with institutions as silver/platinum memberships), grants, and workshop fees. The vast majority of organizational 
+memberships are joint between SWC and DC, so institutional members are not likely to experience substantially different 
+interactions with The Carpentries than with SWC and DC as they currently exist. 
+* The Carpentries would operate under a unified budget initially sheltered under NumFOCUS, but will initiate planning 
+for emergence as an independent 501(c)(3) non-profit organization registered in the USA.
```

Comment:
We may not necessarily be planning to be our own 501(c)3 in the short term. I think we could leave this at 'initially sheltered under NumFOCUS.'